File: University-Projects/intro-to-programming/Python/Workshops/Workshop 1/Convert km to ms.py
# Workshop 1 Code
# Make a converter from km/h to m/s

print("Please type your speed in Kilometers per hour\n")
km = int(input())
m = (km * 1000) / 3600
mps = round(m, 2)
print("you ae travelling at " + str(mps) + " meters per second!\n")
print("Thats so quick dude\n")
print("You should really slow down\n")

File: University-Projects/intro-to-programming/C/Workshops/Workshop 1/Convert km to ms.c
#include <stdio.h>


// Workshop 1 Code 
// Make a Converter from km/h to m/s



int main(void)
{

//Declare Variables	
	
	int km;
	int m;
	
// Ask For INPUT
	
	printf("Please type your speed in Kilometers per hour\n");
	
	scanf("%d", &km);
	
// Do MATH
	
	m = km*1000/3600;
	
// OUTPUT result
	
	printf("You are travelling at %d meters per second \n", m); 
	
// Give Life Advice
	
	printf("Thats so quick dude\n");
	printf("You should really slow down\n");
	
	return 0;
}